pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -655,6 +657,7 @@ func handleVaultStatusImpl(ctxArg interface{}, key string,
 	ctx := ctxArg.(*nodeagentContext)
 	vault := statusArg.(types.VaultStatus)
 
+	ctx.vaultmgrReported = true
 	if ctx.vaultTestStartTime == 0 {
 		//First update from vaultmgr, record it as test start time
```

Comment:
The more logical would be if !vaultMgrReported { vaultMgrReported = true; record test start time}